src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -272,6 +309,8 @@ class ContentSidebar extends PureComponent<Props, State> {
                                 appElement={this.appElement}
                                 rootElement={this.rootElement}
                                 onInteraction={this.onInteraction}
+                                onDescriptionChange={file.can_rename && this.onDescriptionChange}
+                                descriptionTextareaProps={{ maxlength: '255' }}
```

Comment:
This should not be here, should only be local to DetailsSidebar

Reply:
Cool. I left it there in case we wanted to handle the error state (we would pass in `error` with some string), but that could always be passed down separately.